admin/services/pin-generation.service.js

```diff
@@ -62,11 +62,9 @@ pinGenerationService.getPupils = async (dfeNumber, sortField, sortDirection) =>
  * @param pupils
  * @returns {Promise<*>}
  */
-pinGenerationService.filterGroups = async (schoolId, pupils) => {
-  let pupilIds = ''
-  if (pupils.length > 0) {
-    pupils.map(p => { if (p.group_id) pupilIds += "'" + p.group_id + "', " })
-    pupilIds = pupilIds.length > 0 ? pupilIds.substr(0, pupilIds.length - 2) : ''
+pinGenerationService.filterGroups = async (schoolId, pupilIds) => {
+  if (pupilIds.length < 1) {
```

Comment:
what does return false do in this context?

Reply:
Replaced with empty array